Automattic/wp-calypso
# Cloudflare Analytics: Fix Plans Displayed after Nudge

#### Changes proposed in this Pull Request

* Registers the Cloudflare Analytics feature and marks them as a feature for Premium plans (and above) 
* Ensures the nudge redirects to the Plans page with just those plans

#### Testing instructions

Presently, it's a little unclear that Cloudflare Analytics is a feature on Premium plans and higher, especially from this upsell nudge.

<img width="1118" alt="Screenshot 2021-03-27 at 10 22 23" src="https://user-images.githubusercontent.com/43215253/112717716-5a16d480-8ee6-11eb-8fef-222e93e220a4.png">

In order to test this change, click that nudge (on the Stats page) and the one on `/marketing/traffic/site`.

<img width="1109" alt="Screenshot 2021-03-27 at 10 23 28" src="https://user-images.githubusercontent.com/43215253/112717742-7c105700-8ee6-11eb-86d9-9a376e86da61.png">

Now when you click those nudges, you should only see the plans which are compatible with this feature, and the Personal plan shouldn't appear (see #41044 for more details). This is consistent with what happens if you click the Google Analytics nudge on the Marketing page located right below the nudge shown above.

<img width="1136" alt="Screenshot 2021-03-27 at 10 23 42" src="https://user-images.githubusercontent.com/43215253/112717752-8599bf00-8ee6-11eb-895a-76f77672ec8d.png">

cc @blackjackkent, @donlair, @sixhours
